### PR TITLE
Add option to run tests from t keypress

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

**Comment by garyb** [comment]
I'm going to open another PR in a sec that allows extra source globs to be included too, so if you're happy with this it may be worth holding off until that comes in too before releasing.

**Comment by kritzcreek** [comment]
Awesome! Thanks a lot. What do you think of a keybinding/option/toggle `T` that runs your test command whenever the rebuild passes?
Not that it should go into this PR but I think it might be a useful addition

File: src/Pscid/Options.purs [hunk 1]
```diff
@@ -39,10 +39,16 @@ npmCmd = if platform == Win32 then "npm.cmd" else "npm"
...
+  hbs ← hasNamedScript "build"
+  hts ← hasNamedScript "test"
+  pure $ defaultOptions
+    { buildCommand = mkCommand hbs "build"
+    , testCommand = mkCommand hts "test"
+    }
+
+mkCommand ∷ Boolean → String → String
+mkCommand true cmd = npmCmd <> " run -s " <> cmd
+mkCommand false cmd = pulpCmd <> " run -s " <> cmd
```
Comment (kritzcreek) [inline src/Pscid/Options.purs]: strip out the "run -s" here?

Comment (garyb) [inline src/Pscid/Options.purs]: Oops, good catch!

**Comment by garyb** [comment]
Updated, and reduced some duplication.

**Comment by garyb** [comment]
And yeah! Auto-test with toggle key sounds like a useful feature, maybe keypresses to toggle both the `--build` and `--test` flags would be good even.

**Comment by kritzcreek** [comment]
I'm not even using the --build flag. I think it's a leftover of me learning yargs^^ Not sure if it even makes sense to turn that off... :D

**Comment by kritzcreek** [comment]
I'll take this as it is, and we can still add the toggles later :) Thanks 👍

**Pull request merged**